industry_benchmarks/utils/plan_rbfe_network.py

```diff
@@ -0,0 +1,243 @@
+import string
+import click
+import pathlib
+import logging
+import warnings
+import os
+from functools import partial
+from openff.units import unit
+import openfe
+from openfe.protocols.openmm_rfe.equil_rfe_methods import RelativeHybridTopologyProtocol
+from rdkit import Chem
+import kartograf
+from openff.toolkit import (
+    RDKitToolkitWrapper, AmberToolsToolkitWrapper
+)
+from openff.toolkit.utils.toolkit_registry import (
+    toolkit_registry_manager, ToolkitRegistry
+)
+
+logger = logging.getLogger(__name__)
+
+amber_rdkit = ToolkitRegistry([RDKitToolkitWrapper(), AmberToolsToolkitWrapper()])
+
+
+def gen_charges(smc):
+    """
+        Generate AM1BCC partial charges for a SmallMoleculeComponent using
+        the input conformer and antechamber as backend.
+    """
+    offmol = smc.to_openff()
+    with toolkit_registry_manager(amber_rdkit):
+        offmol.assign_partial_charges(partial_charge_method="am1bcc")
+    return openfe.SmallMoleculeComponent.from_openff(offmol)
+
+
+def gen_ligand_network(smcs):
+    """
+    Creates the Lomap ligand network using the KartografAtomMapper and
+    the Lomap scorer.
+
+    Parameters
+    ----------
+    smcs : list[SmallMoleculeComponents]
+      List of SmallMoleculeComponents of the ligands.
+
+    Returns
+    -------
+    openfe.LigandNetwork
+      The Lomap generated LigandNetwork.
+    """
+    mapper = kartograf.KartografAtomMapper(atom_map_hydrogens=True)
```

Comment:
from today's call - should we pin the defaults?

Reply:
Added this!